Gemfile.lock

```diff
@@ -669,7 +669,7 @@ DEPENDENCIES
   will_paginate
 
 RUBY VERSION
-   ruby 2.5.5p157
+   ruby 2.6.3p62
```

Comment:
This is unexpected. Why isn't it 2.6.6p146?

Reply:
Good catch, thanks